partner_statement/report/outstanding_statement.py

```diff
@@ -71,6 +72,10 @@ def _display_outstanding_lines_sql_q1(self, partners, date_end, account_type):
                         pc.max_date <= %(date_end)s) OR
                     (pd.id IS NULL AND pc.id IS NULL)
                 ) AND l.date <= %(date_end)s AND m.state IN ('posted')
+                AND CASE
+                    WHEN %(show_only_overdue)s THEN l.date_maturity <= %(date_end)s
```

Comment:
```suggestion
                    WHEN %(show_only_overdue)s THEN COALESCE(l.date_maturity, l.date) <= %(date_end)s
```
could you apply this to all?

Reply:
Makes sense because the "Date due" shown is actually `date` when `date_maturity` is null:
https://github.com/OCA/account-financial-reporting/blob/a8b5360f696d9d1a2df90ea7f210604e329d6ae3/partner_statement/report/outstanding_statement.py#L45-L48
Thanks!
Please check if it's ok now